go.mod

```diff
@@ -1,10 +1,13 @@
 module github.com/terraform-providers/terraform-provider-fastly
 
 require (
-	github.com/ajg/form v0.0.0-20160822230020-523a5da1a92f // indirect
-	github.com/dnaeon/go-vcr v1.0.1 // indirect
-	github.com/fastly/go-fastly v1.2.1
-	github.com/google/jsonapi v0.0.0-20180313013858-2dcc18f43696 // indirect
-	github.com/hashicorp/go-cleanhttp v0.5.1
+	github.com/ajg/form v1.5.1 // indirect
+	github.com/creack/pty v1.1.9 // indirect
+	github.com/fastly/go-fastly v1.4.0
+	github.com/google/jsonapi v0.0.0-20181016150055-d0428f63eb51 // indirect
 	github.com/hashicorp/terraform-plugin-sdk v1.1.0
+	github.com/kr/pretty v0.2.0 // indirect
+	github.com/kr/pty v1.1.8 // indirect
+	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
+	gopkg.in/yaml.v2 v2.2.7 // indirect
```

Comment:
Please provide me the `go mod` commands you have used in order to upgrade the `go-fastly` dependency. As recommended by the [docs](https://github.com/golang/go/wiki/Modules#how-to-upgrade-and-downgrade-dependencies) a common starting point when upgrading a dependency is to do `go get <dependency>` or `go get <dependency>@latest`.

When you upgrade dependencies it is also always good to run the integration test suite and to document the results in the PR like [here](https://github.com/terraform-providers/terraform-provider-fastly/pull/191) for example (please keep in mind that it takes about 30 to 45 minutes to run the suite).

Reply:
I originally used `go get -u github.com/fastly/go-fastly`. I have upgraded go-fastly again without the `-u` flag as per your advice in 1cab2bf823ad7885cd882d92e79201b2e20b00b5